tools/src/main/java/org/apache/kafka/tools/TransactionalMessageCopier.java

```diff
@@ -264,12 +272,14 @@ public void run() {
         });
 
         try {
+            Random random = new Random(System.currentTimeMillis());
```

Comment:
Ack. We can use the default. I doubt we will make much use of the seed if it were printed. The logs would have to be pretty bad to need to resort to generating the sequence of commit/abort calls.